client/my-sites/google-my-business/stats/chart.js

```diff
@@ -206,8 +206,34 @@ class GoogleMyBusinessStatsChart extends Component {
 		return sumBy( flatten( transformedData ), 'value' ) === 0;
 	}
 
+	renderChartNotice( isError = false ) {
+		const { translate } = this.props;
+
+		const emptyText = translate( 'No activity this period', {
+			context: 'Message on empty bar chart in Stats',
+			comment: 'Should be limited to 32 characters to prevent wrapping',
+		} );
+
+		const errorText = translate( 'Error loading data', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
@cbauerman Could you please convert this to a comment?